modules/apps/frontend-js/frontend-js-web/npmscripts.config.js

```diff
@@ -13,6 +13,15 @@
  */
 
 module.exports = {
+	bridges: [
```

Comment:
See the comments in the commit of this change for a rationale on adding this.

Reply:
Are these bridges the same concept of bridges as before or something different?

Reply:
They are different. 

The previous bridges connected an AMD endpoint to another AMD endpoint. For example:

`my-project@1.0.0/my/legacy/module` -> `my-project@1.0.0/my/new/shiny/module`

These new bridges are a temporary thing (unless it is proven useful for customer migrations in the future) to bridge AMD endpoints to points inside webpack bundles (which otherwise are opaque). For example:

`my-project@1.0.0/my/former/amd/module` -> (some place inside main.bundle.js generated by webpack)

Hope that makes it clearer.

Reply:
Currently we need them to avoid breaking AMD code when a module is migrated to webpack. Say we have migrated `frontend-js-web` completely to `webpack` (i.e. we are not invoking the bundler at all). But some other projects from the portal still want to invoke `frontend-js-web` through the AMD loader so if we don't provide those endpoints the portal will break.

It's the compatibility layer we didn't want to create when all this began.